### PR TITLE
Fix tag names

### PR DESCRIPTION
**Pull request by aug24**

### CODE REVIEW

File: src/test/scala/com/gu/configraun/models/ConfigurationTest.scala [hunk 1]
```diff
@@ -6,20 +6,36 @@ import org.scalatest.{FreeSpec, Matchers}
...
+  "put seq string config in, get seq string config out" in {
+    val conf = Configuration.empty.withList("test1", List("test2", "test3")).getAsList("test1")
+    conf.right.toOption.get shouldBe List("test2", "test3")
+    conf.left.toOption shouldBe None
   }
 
   "don't put config in, don't get config out" in {
-    Configuration.empty.getAsString("test1").left.get.message.startsWith(s"Parameter with key test1 is not present.") shouldBe true
+    val conf = Configuration.empty
+    conf.getAsString("test1").left.get.message.startsWith(s"Parameter with key test1 is not present.") shouldBe true
```
Comment (adamnfish) [inline src/test/scala/com/gu/configraun/models/ConfigurationTest.scala]: One assertion per test is better, if possible!

Comment (adamnfish) [inline src/test/scala/com/gu/configraun/models/ConfigurationTest.scala]: Also, better to use scalatest matchers than to test startswith using a collection operation and then scalatest that the result is true. It'll be more readable and give you better error messages.

```scala
str should startWith ("prefix")
```
http://www.scalatest.org/user_guide/using_matchers#stringsAndArraysAsCollections

File: src/main/scala/com/gu/configraun/Configraun.scala [hunk 2]
```diff
@@ -9,11 +9,11 @@ import com.gu.configraun.models._
 object Configraun {
 
   def loadConfig(implicit client: AWSSimpleSystemsManagement): Either[ConfigraunError, Configuration] = for {
-    stackName <- AwsInstanceTags("stack")
+    stackName <- AwsInstanceTags("Stack")
```
Comment (adamnfish) [inline src/main/scala/com/gu/configraun/Configraun.scala:12]: Argh, good catch!

File: src/main/scala/com/gu/configraun/models/Configuration.scala [hunk 3]
```diff
@@ -13,7 +13,7 @@ case class Configuration(underlying: Map[String, Param]) {
       case Some(ListParam(param)) => Left(ParamNotOfTypeError(s"Parameter with key $key is not of type String. Try calling .getAsList.", new UnsupportedOperationException))
       case None => Left(ParamNotExistError(s"Parameter with key $key is not present.\n" +
         s"Consider adding with " +
-        s"ssm put-parameter --name '/mystack/myapp/PROD/$key' --value 'yourvalue' --type String", new UnsupportedOperationException))
+        s"aws ssm put-parameter --name '/mystack/myapp/PROD$key' --value 'yourvalue' --type String", new UnsupportedOperationException))
```
Comment (adamnfish) [inline src/main/scala/com/gu/configraun/models/Configuration.scala:16]: :+1:

**Review by adamnfish — APPROVED** [review]
I don't know the codebase well, but LGTM

**Comment by LATaylor-guardian** [comment]
👍

**Pull request merged**